.github/workflows/Automation Master Legacy Workflow.yml

```diff
@@ -279,40 +279,125 @@ jobs:
             exit(1)
         shell: python
 
+  artifact_creation:
+      name: Artifact Creation
+      runs-on: ubuntu-latest
+      steps:
+        # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+        - uses: actions/checkout@v4
+          with:
+            fetch-depth: 0
+  
+        - name: Initialize
+          run: |
+            echo "workspace" ${{ github.workspace }}
+            echo "ref name" ${{ inputs.referenceName }}
+            echo "run number" ${{ inputs.runNumber }}
+            echo "ref type" ${{ inputs.referenceType }}
+            echo "repository" ${{ inputs.repository }}
+  
```

Comment:
Can be removed as GitHub shows the inputs anyway: 
![image](https://github.com/user-attachments/assets/1f6e8fc2-01af-43f1-8e19-fd5e34324be0)

Reply:
Same can be done for the SDK workflow